week-1-practise/level-2/1-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(num) {
+return num * 0.2
```

Comment:
Very good! Remember that indentation though.
```suggestion
    return num * 0.2
```